pkg/profiler/profiler.go

```diff
@@ -132,6 +125,10 @@ func (p *CgroupProfiler) Stop() {
 	p.mtx.Lock()
 	defer p.mtx.Unlock()
 	level.Debug(p.logger).Log("msg", "stopping cgroup profiler")
+	err := p.missingStacks.DeleteLabelValues()
+	if !err {
+		fmt.Errorf("error deleting profiles")
```

Comment:
also, like, find a better way to do this.

Reply:
What do you mean with better? This should be totally fine to do :slightly_smiling_face: 
```suggestion
		return fmt.Errorf("error deleting profiles: %w", err)
```